init.sh

```diff
@@ -24,82 +27,91 @@ make install
 evmosd config keyring-backend $KEYRING
 evmosd config chain-id $CHAINID
 
-# if $KEY exists it should be deleted
-evmosd keys add $KEY --keyring-backend $KEYRING --algo $KEYALGO
+# If keys exist they should be deleted
+for KEY in "${KEYS[@]}"
+do
+  evmosd keys add $KEY --keyring-backend $KEYRING --algo $KEYALGO --home $homedir
+done
 
 # Set moniker and chain-id for Evmos (Moniker can be anything, chain-id must be an integer)
-evmosd init $MONIKER --chain-id $CHAINID
+evmosd init $MONIKER -o --chain-id $CHAINID --home $homedir
 
 # Change parameter token denominations to aevmos
-cat $HOME/.evmosd/config/genesis.json | jq '.app_state["staking"]["params"]["bond_denom"]="aevmos"' > $HOME/.evmosd/config/tmp_genesis.json && mv $HOME/.evmosd/config/tmp_genesis.json $HOME/.evmosd/config/genesis.json
-cat $HOME/.evmosd/config/genesis.json | jq '.app_state["crisis"]["constant_fee"]["denom"]="aevmos"' > $HOME/.evmosd/config/tmp_genesis.json && mv $HOME/.evmosd/config/tmp_genesis.json $HOME/.evmosd/config/genesis.json
-cat $HOME/.evmosd/config/genesis.json | jq '.app_state["gov"]["deposit_params"]["min_deposit"][0]["denom"]="aevmos"' > $HOME/.evmosd/config/tmp_genesis.json && mv $HOME/.evmosd/config/tmp_genesis.json $HOME/.evmosd/config/genesis.json
-cat $HOME/.evmosd/config/genesis.json | jq '.app_state["evm"]["params"]["evm_denom"]="aevmos"' > $HOME/.evmosd/config/tmp_genesis.json && mv $HOME/.evmosd/config/tmp_genesis.json $HOME/.evmosd/config/genesis.json
-cat $HOME/.evmosd/config/genesis.json | jq '.app_state["inflation"]["params"]["mint_denom"]="aevmos"' > $HOME/.evmosd/config/tmp_genesis.json && mv $HOME/.evmosd/config/tmp_genesis.json $HOME/.evmosd/config/genesis.json
+cat $homedir/config/genesis.json | jq '.app_state["staking"]["params"]["bond_denom"]="aevmos"' > $homedir/config/tmp_genesis.json && mv $homedir/config/tmp_genesis.json $homedir/config/genesis.json
```

Comment:
This `$homedir/config/genesis.json` and this `$homedir/config/tmp_genesis.json` are long and being repeated multiple times, how do you think about make variables GENESIS and TMP_GENESIS so the line would looks shorter and easier to read?

Reply:
Good point, thanks!